### ISSUE TITLE
Sell Menu Not Showing

### ISSUE THREAD

**Issue by IVIcToddy** [lenzh/Lenzh_chopshop#32]
sell menu is invisible bit if i randomly press enter it still allows me to sell the items i have chopped its just not showing me the actual menu

**Comment by davem1988** [comment]
I have the Exact same problems, when looking in my client console it told me the problem is with "esx_menu_default:nui"

Client Console (F8)
![Capture3](https://user-images.githubusercontent.com/62862723/120904694-8902a280-c61b-11eb-88f0-417f07ef267d.JPG)


cl_main.lua
![Capture](https://user-images.githubusercontent.com/62862723/120904695-899b3900-c61b-11eb-8ed2-ecab30b93c1c.JPG)


sv_main.lua
![Capture2](https://user-images.githubusercontent.com/62862723/120904697-899b3900-c61b-11eb-8aed-66a5236ed606.JPG)

**Comment by IVIcToddy** [comment]
Yeah it’s the same for me, Have you got any idea how to fix it because I’ve been trying to correct it for a while now and still no luck 😂 I’m quite new to the whole scripting and server hosting so I’m not exactly the best at fixing things like this 😂

**Comment by davem1988** [comment]
> Yeah it’s the same for me, Have you got any idea how to fix it because I’ve been trying to correct it for a while now and still no luck 😂 I’m quite new to the whole scripting and server hosting so I’m not exactly the best at fixing things like this 😂

No solution for now! But if i found it i'll post it here! 😊

**Comment by heavenlygamer04** [comment]
@davem1988 @IVIcToddy

 i found a fix! rename the folder to "Lenzh_chopshop" instead of "Lenzh_chopshop-2.0"
